docs/iris/src/installing.rst

```diff
@@ -1,6 +1,93 @@
 .. _installing_iris:
 
 Installing Iris
-***************
+===============
 
-.. include:: ../../../INSTALL
+Iris is available using conda for the following platforms:
+
+* Linux 64-bit,
+* Mac OSX 64-bit, and
+* Windows 64-bit.
+
+.. note:: Iris currently supports and is tested against **Python 3.6** and
+          **Python 3.7**.
+
+
+.. _installing_using_conda:
+
+Installing using conda (users)
+------------------------------
+
+To install Iris using conda, you must first download and install conda,
+for example from https://docs.conda.io/en/latest/miniconda.html.
+
+Once conda is installed, you can install Iris using conda with the following
+command::
+
+  conda install -c conda-forge iris
+
+If you wish to run any of the code in the gallery you will also
+need the Iris sample data. This can also be installed using conda::
+
+  conda install -c conda-forge iris-sample-data
+
+Further documentation on using conda and the features it provides can be found
+at https://conda.io/en/latest/index.html.
+
+
+.. _installing_from_source:
+
+Installing from source (devs)
+-----------------------------
+
+The latest Iris source release is available from
+https://github.com/SciTools/iris.
+
+For instructions on how to obtain the Iris project source from GitHub see
+:ref:`forking` and :ref:`set-up-fork` for instructions.
+
+Once conda is installed, you can install Iris using conda and then activate
+it.  The example commands below assume you are in the root directory of your
+local copy of Iris::
+
+  conda env create --file=requirements/ci/iris.yml
+  conda activate iris-dev
+
+The ``requirements/ci/iris.yml`` file used to create the conda environment
+includes **all** the relevant dependencies that you need to **code**, **test**,
+and **build** the documentation.  If you wish to minimise your install you can
+remove any unwanted packages from the requirements file.
+
+.. note:: The ``requirements/ci/iris.yml`` file will always use the latest
+          tested Python version available.  For all Python versions that
+          are supported view the contents of the `requirements/ci`_ directory.
+
+.. _requirements/ci: https://github.com/tkknight/iris/tree/master/requirements/ci
```

Comment:
@tkknight This link with need to change, right?

Reply:
In fact, this URL is causing the travis-ci sphinx link heck to fail

Reply:
Yes I do need to change this (noticed it earlier today but not had time to push an update until now).  However the correct link will still fail the travis linkcheck until the new conda requirements pr (https://github.com/SciTools/iris/pull/3812) is merged to **master**, which is why I kept this PR in draft for the interrim.